api/v1beta2/ocirepository_types.go

```diff
@@ -78,6 +78,13 @@ type OCIRepositorySpec struct {
 	// +optional
 	SecretRef *meta.LocalObjectReference `json:"secretRef,omitempty"`
 
+	// Verify specifies which provider to use to check whether OCI image is authentic.
+	// Implements RFC-0003.
+	// +kubebuilder:validation:Enum=cosign
+	// +kubebuilder:default:=cosign
```

Comment:
```suggestion

```

Verify is a struct which can't have a default, the default is nil.

Reply:
thx, fixed.